textgrid/textgrid.py

```diff
@@ -314,6 +314,14 @@ def __init__(self, name=None, minTime=0., maxTime=None):
         self.maxTime = maxTime
         self.points = []
 
+    def __eq__(self, other):
+        if not hasattr(other, 'points'):
+            return False
+        elif all(a == b for a, b in zip(other.points, self.points)):
```

Comment:
I would do:

else:
    return all(a == b for a, b in zip(self.points, other.points))

That eliminates the true/false switch and I think it's natural to have `self` before `other`.

Reply:
Done.